server/api/cycleData/table/persistNodeValues.ts

```diff
@@ -24,6 +24,7 @@ export const persistNodeValues = async (req: CycleDataRequest<never, NodesBody>,
           countryIso,
           assessment,
           cycle,
+          section,
```

Comment:
pass the parameter as sectionName. this should be ```sectionName:section```
in CycleDataController.persistNodeValue and all related method use sectionName